About.html

```diff
@@ -0,0 +1,115 @@
+          <ul class="nav-ul">
+            <li class="nav-li"><a href="index.html">Home</a></li>
+            <li class="nav-li">Program</li>
+            <li class="nav-li">Join</li>
+            <li class="nav-li">Sponsors</li>
+            <li class="nav-li news">News</li>
+          </ul>
+        </nav>
+        <section class="aboutpg">
+        <div>
+        <h1 class="topic">2023 AGRICUTURAL SUMMIT THEME:SUSTAINNABLE LIVING</h1>
+        </div>
+        <div class="details">
+        <p class="pg">The Global Agricultural Summit serves as a crucial platform that unites a diverse community of experts, academics, and activists who form the creative commons affiliate network. With the aim of addressing pressing agricultural challenges, this influential event takes place in a different country every two years. In each edition, the summit brings together thought leaders and stakeholders from various backgrounds, fostering collaboration and knowledge exchange.</p>
+        <p class="contactus">Please contact us per Email for any futher questions</p>
+            <p>About AGRICULTURAL SUMMIT 2023</p>
+            <a href="" class="d-link">agrisummit@gmail.com</a>
+    </div>
+    </section>
+    <section class="logoSection">
+        <h2 class="agro-theme">Agricultural Summit 2023 Logo</h2>
+    <hr>
+    <p class="agro-talk">The Logo of the 2023 agricultural summit was determined during the farm stead display
+        <br>through the logo competition from 8 August to 7 January.
+    </p>
+   <div class="agri-div">
+       <img src="./images/LOGO.jpeg" alt="agri-logo" class="b-logo">
+   </div>
+    <hr>
+    <div class="past-events">
+    <h4>See past Agric Sumits</h4>
+    <p class="past-E">Take a look at the two Agricultural summit at other world locations
+        <br>in the Hawaii reefs and in Portugal</p>
+        <div class="event-container">
+            <div class="event-d event-1">
+                <div class="event-cover">
+                        <p class="event-p">
+                            2022 <br>
+                            PRODUCTION IMPROVMENTS THROUGH PROPER FERTILIZERS
+                        </p>
+                </div>
+            </div>
+            <div class="event-d event-2">
+                <div class="event-cover">
+                        <p class="event-p">
+                            2020 <br>
+                            CROP PEST ERRADICATION AND CONTROL GLOBALLY AND LOW
+                        </p>
+                </div>
+            </div>
+</div>
+    </section>
+    <section class="partners-section">
+        <div class="flex">
+          <h4 class="partners-head">Partners</h4>
+        </div>
+        <div class="line"></div>
+        <div class="partners">
+          <img src="images/google.png" alt="">
+          <img src="images/EA-SPORT.png" alt="">
+          <img src="images/samsung_2022-logo.png" alt="">
+          <img src="images/NASA.png" alt="" class="stub">
```

Comment:
- [ ] According to your previous reviewer  In the mobile design of the About Page, the Partners section is present but it is absent in the desktop version of the About Page. Kindly implement this detail in the design of the mobile version and make the Partners section present. 💻
- [ ] [see previous review here](https://github.com/Danny-imo/Capstone-project/pull/2#discussion_r1252133424)